AssemblyToProcessNullableRefTypes/AssemblyToProcessNullableRefTypes.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Optimize>true</Optimize>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <Nullable>enable</Nullable>
+    <!--<DisableFody>true</DisableFody>-->
```

Comment:
This should probably not be commented out. Weavers are usually not run as part of the build, the unit tests are supposed to run them.
The `WeaverFiles` item and reference to `Fody` should be removed as well. The `FodyWeavers` files are also not needed.

Reply:
I tried uncommenting that line, but then a bunch of tests start failing

Reply:
@ltrzesniewski this is a can, not a rule. Writing test this way is much more reliable, robust and convenient.